src/components/UTIL_HtmlOutput.component

```diff
@@ -0,0 +1,6 @@
+<apex:component layout="none" id="util_htmloutput" controller="UTIL_HtmlOutput_CTRL">
```

Comment:
What would you think about `UTIL_HtmlLabelOutput` for a filename?

Reply:
Thanks team, I have a new idea I will put together and bug you again in a bit.